index.d.ts

```diff
@@ -73,6 +73,7 @@ declare namespace pathToRegexp {
      * Function for encoding input strings for output.
      */
     encode?: (value: string, token: Key) => string;
+    skipMatch?: boolean;
```

Comment:
Would you mind adding some documentation here and mirroring it to the README?

Reply:
Maybe calling it `noValidate` or something like that? Do you feel like that's any clearer about intent?

Reply:
I rename `skipMatch` to `noValidate` and add the documentation.